meta-fobnail-distro/recipes-fobnail/qcobr/qcbor_git.bb

```diff
@@ -0,0 +1,36 @@
+LICENSE = "CLOSED"
+LIC_FILES_CHKSUM = ""
+
+SRC_URI = "git://github.com/laurencelundblade/QCBOR;protocol=https;branch=master"
+
+PV = "1.0+git${SRCPV}"
+SRCREV = "44754f738c6534a4304a83d4c6e97b3d3193d887"
+
+S = "${WORKDIR}/git"
+
+FILES:${PN} += " \
+    /usr/local/include/qcbor \
```

Comment:
@kotylamichal we install files in `/usr/local/include` or `$includedir`?

Reply:
fixed